ResXManager.Model/ResourceManagerExtensions.cs

```diff
@@ -60,7 +72,47 @@ public static IList<ProjectFile> GetAllSourceFiles([NotNull] this DirectoryInfo
                 foreach (var file in directoryFiles)
                 {
                     Contract.Assume(file != null);
+                    string @namespace = string.Empty;
+                    if (project != null)
+                    {
+                        //Get the ProjectFile Directory and make sure that every file seperator is the same
+                        string projectDirectory = project.DirectoryName?.Replace(@"\\", @"\") ?? string.Empty;

```

Comment:
Better use Uri.MakeRelativeUri to calculate relative file paths, it does not crash for files outside the project structure.